chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -1955,14 +1954,14 @@ async def get_coin_records(coin_ids: Collection[bytes32]) -> List[CoinRecord]:
         # expect both test items to get included in the block.
         # NOTE: The expected block cost is smaller than the sum of items' costs
         # because of the spend bundle aggregation that creates the block bundle.
-        (TEST_FILL_RATE_ITEM_COST * 2, 2, TEST_FILL_RATE_ITEM_COST * 2 - 61_829),
+        (TEST_FILL_RATE_ITEM_COST * 2, 2, TEST_FILL_RATE_ITEM_COST * 2 - 156_020),
```

Comment:
this difference increasing suggests that the "effect of compression" was not avoided, doesn't it?
Do you understand why this changes?